README.md

```diff
@@ -1,148 +1,91 @@
-# Phoenics
+# Data-driven experiment planning strategies 
 
-[![Build Status](https://travis-ci.com/FlorianHase/phoenics.svg?token=rULvnKYmWdFF3JqQBVVW&branch=master)](https://travis-ci.com/FlorianHase/phoenics)
+This repository offers data-driven experiment planning strategies following principles of Bayesian optimization in combination with Bayesian kernel density estimation to perform cost-effective global optimization on physical experiments and/or resource-intensive computations.
 
-Phoenics is an open source optimization algorithm combining ideas from Bayesian optimization with Bayesian Kernel Density estimation [1]. It performs global optimization on expensive to evaluate objectives, such as physical experiments or demanding computations.
+The mathematical formulation of this algorithms provided in this repository are based on Phoenics [1]. Among several functionalities, Phoenics targets data-driven experiment planning tasks particularly in the context of chemistry and materials science. It is possible to use Phoenics for the rapid identification of optimal continuous process parameters for physical experiments or computations. Phoenics natively supports sequential and batch-wise optimizations. 
 
-Check out the `examples` folder for detailed descriptions and code examples for:
+![continuous_integration](https://github.com/chemos-inc-dev/phoenics/workflows/Continuous%20integration/badge.svg)
+[![codecov](https://codecov.io/gh/chemos-inc-dev/phoenics/branch/dev/graph/badge.svg?token=anTJ0HXjI6)](https://codecov.io/gh/chemos-inc-dev/phoenics)
+![github_license](https://img.shields.io/github/license/chemos-inc/phoenics)
+![github_issues](https://img.shields.io/github/issues/chemos-inc/phoenics)
+![pypi_version](https://img.shields.io/pypi/v/phoenics)
+
+Check out our `examples` for a few highlights and use cases of Phoenics.
 
 | Example | Link |
 |:--------|:-----|
 | Sequential optimization           |  [examples/optimization_sequential](https://github.com/chemos-inc/phoenics/tree/master/examples/optimization_sequential)  |
 
 
-# Installation
-
-## Environment (Optional)
 
-We suggest to create a new environment for the installation of Phoenics to avoid any compatibility issues with pre-existing python packages. This is optional, but recommended.
+### Getting started
 
-This can be done with Venv or Anaconda.
+Phoenics is developed for Linux based systems. We also support MacOS and Windows, but support for these platforms is experimental at the moment. Although we hope that you will not encounter any issues we would appreciate if you run into any of them.
```

Comment:
This sentence is strange to me: `Although we hope that you will not encounter any issues we would appreciate if you run into any of them.`
Do we want the user to run into issues?

Reply:
Actually no - something about "please contact us" is missing. I'll add that to the readme - thanks for catching that!

Reply:
That makes sense :)